### ISSUE TITLE
[ERROR] [ydlidar_ros2_driver_node-1]: process has died, exit code -6

### ISSUE THREAD

**Issue by hfcaio** [YDLIDAR/ydlidar_ros2_driver#29]
when i run `ros2 launch ydlidar_ros2_driver ydlidar_launch.py` and `ros2 launch ydlidar_ros2_driver ydlidar_launch_view.py` both of then give me this error

![image](https://github.com/YDLIDAR/ydlidar_ros2_driver/assets/83253047/a96efa50-e6ed-46f5-afb0-cbe54e3edf60)

**Comment by Ahtisham123Ahmed** [comment]
@hfcaio did you find out solution?

**Comment by jungwoonghyun** [comment]
@hfcaio Hi did you find out solution?

**Comment by jungwoonghyun** [comment]
@Ahtisham123Ahmed Did you find out solution?

**Comment by Ahtisham123Ahmed** [comment]
> @Ahtisham123Ahmed Did you find out solution?

I didnt remember the issue exactly but i figure out the problem and use ydlidarx3pro for my robot

**Comment by jungwoonghyun** [comment]
@Ahtisham123Ahmed Anyway there is some way to fix it. Thanks to let me know

**Comment by joIcecube** [comment]
Hi everyone. any update ? thx all

**Comment by Ahtisham123Ahmed** [comment]
> Hi everyone. any update ? thx all

@joIcecube Yes, I used the YDLidar X3 Pro for my robot. I think I fixed the issue. I can share the file for launching YDLidar and some important settings. There were some issues in the file that needed to be updated.
